drivers/hmis/app/graphql/mutations/delete_assessment.rb

```diff
@@ -0,0 +1,21 @@
+module Mutations
+  class DeleteAssessment < BaseMutation
+    argument :id, ID, required: true
+
+    field :assessment, Types::HmisSchema::Assessment, null: true
+
+    def resolve(id:)
+      assessment = Hmis::Hud::Assessment.viewable_by(current_user).find_by(id: id)
+      errors = []
+
+      errors << HmisErrors::Error.new(:id, :not_found) unless assessment.present?
+
+      assessment.destroy! if assessment.present?
+
+      return {
+        assessment: assessment,
+        errors: errors,
+      }
+    end
```

Comment:
This is the default delete behavior, so we can use the helper for it. It should also be checking the editable scope.

```
def resolve(id:)
      record = Hmis::Hud::Assessment.editable_by(current_user).find_by(id: id)
      default_delete_record(record: record, field_name: :assessment)
    end
```

Reply:
@ZachCMP

Reply:
@gigxz Gotcha, makes sense! Fixed in this commit: https://github.com/greenriver/hmis-warehouse/commit/ae8de75d2de590e14ee9a84d54b1711de13442d4